src/main/java/org/broadinstitute/hellbender/dev/tools/walkers/bqsr/BaseRecalibratorDataflow.java

```diff
@@ -193,7 +194,7 @@ protected static String pickOutputTablesPath(boolean remote, String stagingLocat
             final ReadFilter readFilter = BaseRecalibratorWorker.readFilter(header);
             final List<SimpleInterval> intervals = BRAC.intervalArgumentCollection.intervalsSpecified() ? BRAC.intervalArgumentCollection.getIntervals(sequenceDictionary) :
                     IntervalUtils.getAllIntervalsForReference(sequenceDictionary);
-            return new ReadsSource(beforePath, pipeline).getReadPCollection(intervals, ValidationStringency.SILENT)
+            return new ReadsDataflowSource(beforePath, pipeline).getReadPCollection(intervals, ValidationStringency.SILENT)
```

Comment:
@jean-philippe-martin , any ideas for a better name, or is ReadsDataflowSource good enough?